apps/emqx_durable_storage/src/emqx_ds_bitmask_keymapper.erl

```diff
@@ -185,14 +218,19 @@
 %% Note: order of bitsources is important. First element of the list
 %% is mapped to the _least_ significant bits of the key, and the last
 %% element becomes most significant bits.
+%%
+%% Warning: currently the algorithm doesn't handle situations when
+%% parts of a vector element are _reordered_ in the resulting scalar.
```

Comment:
Maybe we could have a simple example explaining this further?

```suggestion
%% Warning: currently the algorithm doesn't handle situations when
%% parts of a vector element are _reordered_ in the resulting scalar.
%% For example, if the same vector coordinate is written to both the most 
%% and the least significant bits of the key.
```
?

Reply:
That's another exception, yes.